docs-src/docs/dataset-refinement/prediction-format.md

```diff
@@ -194,5 +193,118 @@ quaternions [here](../openlabel/openlabel-format/#rotation-of-cuboids).
 }
 ```
 
+### Segmentation bitmap
+
+## Transforming, upscaling, padding and base64-encoding a small color-image to a larger grayscale image using Python PIL
```

Comment:
"###" is a sub-heading to "##". This is on the same level as "Prediction examples", should it be a "####" instead? (I am not 100% sure it's possible to have 4*#)